modules/osdk-scorecard-olm.adoc

```diff
@@ -0,0 +1,217 @@
+----
+spec:
+  install:
+    spec:
+      deployments:
+      - name: memcached-operator
+        spec:
+          ...
+          template:
+            ...
+            spec:
+              containers:
+              ...
+              volumes: <1>
+              - name: scorecard-kubeconfig
+                secret:
+                  secretName: scorecard-kubeconfig
+                  items:
+                  - key: kubeconfig
+                    path: config
+----
+<1> Scorecard `kubeconfig` volume.
+
+. Insert a volume mount and `KUBECONFIG` environment variable into each container
+in your Operator's Deployment:
++
+[source,yaml]
+----
+spec:
+  install:
+    spec:
+      deployments:
+      - name: memcached-operator
+        spec:
+          ...
+          template:
+            ...
+            spec:
+              containers:
+              - name: container1
+                ...
+                volumeMounts: <1>
+                - name: scorecard-kubeconfig
+                  mountPath: /scorecard-secret
+                env: <2>
+                - name: KUBECONFIG
+                  value: /scorecard-secret/config
+              - name: container2 <3>
+                ...
+----
+<1> Scorecard `kubeconfig` volume mount.
+<2> Scorecard `kubeconfig` environment variable.
+<3> Repeat the same for this and all other containers.
+
+. Insert the scorecard proxy container into the Operator's Deployment:
++
+[source,yaml]
+----
+spec:
+  install:
+    spec:
+      deployments:
+      - name: memcached-operator
+        spec:
+          ...
+          template:
+            ...
+            spec:
+              containers:
+              ...
+              - name: scorecard-proxy <1>
+                command:
+                - scorecard-proxy
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.namespace
+                image: quay.io/operator-framework/scorecard-proxy:master
+                imagePullPolicy: Always
+                ports:
+                - name: proxy
+                  containerPort: 8889
+----
+<1> Scorecard proxy container.
+
+[TIP]
+====
+Alternatively, the
+link:https://github.com/operator-framework/community-operators[`community-operators`]
+repository has several bash functions that can perform this procedure for you:
+
+----
+$ curl -Lo csv-manifest-modifiers.sh \
+    https://raw.githubusercontent.com/operator-framework/community-operators/master/scripts/lib/file
+$ . ./csv-manifest-modifiers.sh
+$ create_kubeconfig_secret_file scorecard-secret.yaml "<namespace>" <1>
+$ oc apply -f scorecard-secret.yaml
+$ insert_kubeconfig_volume "<csv_file>" <2>
+$ insert_kubeconfig_secret_mount "<csv_file>"
+$ insert_proxy_container "<csv_file>" "quay.io/operator-framework/scorecard-proxy:master"
+----
+<1> Replace `<namespace>` with the namespace your Operator will deploy in.
```

Comment:
In above comments, this was worded as "was deployed in". Does it need to be future here?